docs/concepts/dotnet-standard-library.md

```diff
@@ -0,0 +1,80 @@
+.NET Standard Library
+=====================
+
+It can be thought of as the next generation of [Portable Class Libraries](https://msdn.microsoft.com/library/gg597391.aspx), although is architecturally different. You can learn more 
```

Comment:
For anyone who doesn't know what a PCL is, this isn't very helpful. The description of .NET Standard Library in the .NET Core topic, though, provides a really good context. It would be good to begin this topic with it.